modules/experimental-layers/package.json

```diff
@@ -25,8 +25,8 @@
   ],
   "scripts": {
     "build-clean": "rm -fr dist dist-es6 && mkdir -p dist dist-es6",
-    "build-es6": "rm -fr dist-es6 && babel src --out-dir dist-es6 --source-maps inline",
-    "build-es5": "rm -fr dist && babel src --out-dir dist --plugins=transform-es2015-modules-commonjs --source-maps inline",
+    "build-es6": "rm -fr dist-es6 && BABEL_ENV=es6 babel src --out-dir dist-es6 --source-maps inline",
+    "build-es5": "rm -fr dist && BABEL_ENV=es5 babel src --out-dir dist --plugins=transform-es2015-modules-commonjs --source-maps inline",
```

Comment:
I think we should remove the plugins here? This file should look exactly like `modules/core/package.json` right?

Another thing I noticed: Lately we have been trying to avoid inline source maps. They add a lot of size to the bundle. Maybe align in that regard as well, since you are changing these files.

Reply:
good catch- looks like its redundant since it's defined here: https://github.com/babel/babel/blob/master/packages/babel-preset-es2015/src/index.js#L79